liquibase-core/src/main/java/liquibase/GlobalConfiguration.java

```diff
@@ -215,6 +216,11 @@ public class GlobalConfiguration implements AutoloadedConfigurations {
         ALWAYS_DROP_INSTEAD_OF_REPLACE = builder.define("alwaysDropInsteadOfReplace", Boolean.class)
                 .setDescription("If true, drop and recreate a view instead of replacing it.")
                 .build();
+
+        SHOW_UPDATE_SUMMARY = builder.define("showUpdateSummary", Boolean.class)
+                .setDescription("If true, show a summary report after an update operation.")
+                .setDefaultValue(true)
```

Comment:
What would be the impact of changing the default output?

Reply:
Nothing consequential.  The summary would not show up.